scripts/import_cldr.py

```diff
@@ -755,14 +755,24 @@ def parse_currency_names(data, tree):
 
 def parse_unit_patterns(data, tree):
     unit_patterns = data.setdefault('unit_patterns', {})
+    compound_patterns = data.setdefault('compound_unit_patterns', {})
     for elem in tree.findall('.//units/unitLength'):
         unit_length_type = elem.attrib['type']
         for unit in elem.findall('unit'):
             unit_type = unit.attrib['type']
+            box = "%s:%s" % (unit_type, unit_length_type)
+            pattern_box = unit_patterns.setdefault(box, {})
             for pattern in unit.findall('unitPattern'):
-                box = unit_type
-                box += ':' + unit_length_type
-                unit_patterns.setdefault(box, {})[pattern.attrib['count']] = text_type(pattern.text)
+                pattern_box[pattern.attrib['count']] = text_type(pattern.text)
+
+            per_unit_pat = unit.find('perUnitPattern')
+            if per_unit_pat is not None:
+                pattern_box['per'] = text_type(per_unit_pat.text)
+
+        for unit in elem.findall('compoundUnit'):
+            unit_type = unit.attrib['type']
+            box = "%s:%s" % (unit_type, unit_length_type)
```

Comment:
Commit 5204709 introduced the colons in the first place, but for alternative forms.
Then 9327e08 reused colons/string concat for lengths (misguidedly, perhaps).

I guess "for no particular reason" is the answer :)